### PR TITLE
Release 1.1.2

### PR DESCRIPTION
**Pull request by Apokalip**
## Release v1.1.2

**Each reviewer needs to check that these conditions are met before approving the PR.**

- [x] Checked that the release is on the correct branch name of the form `release-vX.Y.Z` 
- [x] Added the `L-skip` label and the relevant `release` label to this PR
- [x] Updated the [`CHANGELOG.md`](https://github.com/manta-network/manta-signer/blob/main/CHANGELOG.md)
- [x] Updated the version numbers in the following files:
    - [x] [`Cargo.toml`](https://github.com/manta-network/manta-signer/blob/main/Cargo.toml)
    - [x] [`ui/package.json`](https://github.com/manta-network/manta-signer/blob/main/ui/package.json)
    - [x] [`ui/public/about.html`](https://github.com/manta-network/manta-signer/blob/main/ui/public/about.html)
    - [x] [`ui/src-tauri/Cargo.toml`](https://github.com/manta-network/manta-signer/blob/main/ui/src-tauri/Cargo.toml)
    - [x] [`ui/src-tauri/tauri.conf.json`](https://github.com/manta-network/manta-signer/blob/main/ui/src-tauri/tauri.conf.json)
    - [x] [`www/src/App.tsx`](https://github.com/manta-network/manta-signer/blob/main/www/src/App.tsx)
- [x] Run the testing procedure defined in [`docs/testing-workflow.md`](https://github.com/manta-network/manta-signer/blob/main/docs/testing-workflow.md)

### After the Release

- [ ] Verify that [the signer download site](https://signer.manta.network) works correctly
- [ ] Verify that all jobs to build signer for all operating systems were successful
- [ ] Verify that the previous signer release is prompted for an update upon sign in

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## v1.1.2 2023-03-08
```
Comment (ferrell-code) [inline CHANGELOG.md:6]: Nit, maybe keep empty unreleased section in changelog to keep format

File: Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "manta-signer"
 edition = "2021"
-version = "1.1.1"
+version = "1.1.2"
```
Comment (ferrell-code) [inline Cargo.toml:4]: If following semantic versioning it probably should at least be 1.2.0.

Ok as is if binaries are already set to go

Comment (Apokalip) [reply Cargo.toml:4]: Normally we would chnage the second number with very big changes while the first would be large refactoring.

File: CHANGELOG.md (hunk 1 above)
Comment (Apokalip) [reply CHANGELOG.md:6]: This is only outside of releases, so we know what goes in next release. Check out other releases, they should not have a Unreleased group

**Review by ghzlatarev — APPROVED** [review]

**Review by ferrell-code — APPROVED** [review]

**Pull request merged**